Tetfuza/TetfuzaBackend/Interfaces/IDisplay.cs

```diff
@@ -0,0 +1,15 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Tetfuza.Interfaces
+{
+    public interface IDisplay
+    {
+        public void GameScreen(TetfuzaBackend gameState);
+        public int DrawMenu(string prompt);
+        public void WriteText(string text);
+        public void ClearScreen();
+        public void RedrawFrame();
+    }
+}
```

Comment:
IDisplay has all the necessary components required to display the game. It probably doesn't need WriteText, now that I think about it...

Reply:
Should it have CenterText()?

Reply:
WriteText() currently has the capabilities of CenterText(), but this will be changing soon. I'm starting to think instead of having a method to only center the text, we should instead have the ability to specify x and y coordinates to write text to. I'm going to need to work this out a bit, but it should allow us to fit the whole game into the default console window size.